src/WrkLst/DocxMustache/DocxMustache.php

```diff
@@ -485,7 +499,7 @@ public function SaveAsPdf()
         } else {
             $path_parts = pathinfo($this->StoragePath($this->local_path.$this->template_file_name));
 
-            return $this->StoragePath($this->local_path.$path_parts['filename'].'pdf');
+            return $this->StoragePath($this->local_path.$path_parts['filename'].'.pdf');
```

Comment:
this seems like a breaking change. could you explain? no breaking changes please. or tackle them in a separate issue if you feel this is something that needs to be changed.

Reply:
sorry that one slipped through. In the path the dot between filename and extension was missing

Reply:
I can remove it from this PR if you want me to.

Reply:
the class is used in production, so any PR should have no breaking changes, or they need to be documented. Please go through your PR and make sure it contains no refactoring leading to breaking changes, as that will lead to problems in the production environments and have to be saved for a major version.